.gitignore

```diff
@@ -0,0 +1,13 @@
+.vagrant/machines/default/virtualbox/action_provision
+.vagrant/machines/default/virtualbox/action_set_name
+.vagrant/machines/default/virtualbox/box_meta
+.vagrant/machines/default/virtualbox/creator_uid
+.vagrant/machines/default/virtualbox/id
+.vagrant/machines/default/virtualbox/index_uuid
+.vagrant/machines/default/virtualbox/private_key
+.vagrant/machines/default/virtualbox/synced_folders
+.vagrant/machines/default/virtualbox/vagrant_cwd
+.vagrant/rgloader/loader.rb
+ubuntu-bionic-18.04-cloudimg-console.log
+GIVIT\GIVIT\__pycache__
```

Comment:
same idea different file/folder type

Reply:
I just want to make sure that I understand:
1. In order to delete pyc files, I have to delete them from my sync directory and push again.
2. from now on, write the whole directory in the gitignore and not each file?
thanks!
sorry for the poor PR
Nadav